packages/react-examples/src/react-charting/LineChart/LineChart.SecondaryYAxis.Example.tsx

```diff
@@ -0,0 +1,154 @@
+import * as React from 'react';
+import { IChartProps, ILineChartProps, LineChart, DataVizPalette } from '@fluentui/react-charting';
+import { getId } from '@fluentui/react';
+
+const data: IChartProps = {
+  chartTitle: 'Line Chart',
+  lineChartData: [
+    {
+      legend: 'From_Legacy_to_O365',
+      data: [
+        {
+          x: new Date('2020-03-03T00:00:00.000Z'),
+          y: 216000,
+        },
+        {
+          x: new Date('2020-03-03T10:00:00.000Z'),
+          y: 218123,
+        },
+        {
+          x: new Date('2020-03-03T11:00:00.000Z'),
+          y: 217124,
+        },
+        {
+          x: new Date('2020-03-04T00:00:00.000Z'),
+          y: 248000,
+        },
+        {
+          x: new Date('2020-03-05T00:00:00.000Z'),
+          y: -252000,
+        },
+        {
+          x: new Date('2020-03-06T00:00:00.000Z'),
+          y: 274000,
+        },
+        {
+          x: new Date('2020-03-07T00:00:00.000Z'),
+          y: -260000,
+        },
+        {
+          x: new Date('2020-03-08T00:00:00.000Z'),
+          y: 304000,
+        },
+        {
+          x: new Date('2020-03-09T00:00:00.000Z'),
+          y: 218000,
+        },
+      ],
+      color: DataVizPalette.color3,
+    },
+    {
+      legend: 'All',
+      data: [
+        {
+          x: new Date('2020-03-03T00:00:00.000Z'),
+          y: 297000,
+        },
+        {
+          x: new Date('2020-03-04T00:00:00.000Z'),
+          y: 284000,
+        },
+        {
+          x: new Date('2020-03-05T00:00:00.000Z'),
+          y: 282000,
+        },
+        {
+          x: new Date('2020-03-06T00:00:00.000Z'),
+          y: -294000,
+        },
+        {
+          x: new Date('2020-03-07T00:00:00.000Z'),
+          y: 224000,
```

Comment:
>224000 [](http://example.com/codeflow?start=13&length=6)

make these values 10x smaller for secondary axis differentiation

Reply:
made the values 1000x smaller